auth/tests/int/attestation.test.ts

```diff
@@ -26,6 +27,19 @@ describe("attestation lambda", () => {
             const response = await loggingDriver.findLogMessageWithRetries({
                 logGroupName: testConfig.authProxyLogGroup,
                 searchString: 'Calling auth proxy',
+                startTime,
+                endTime,
+            })
+
+            expect(response).toBeDefined()
+        })
+
+        it("should access the client secret from secrets manager", async () => {
+            const response = await loggingDriver.findLogMessageWithRetries({
+                logGroupName: testConfig.authProxyLogGroup,
+                searchString: "client secret",
```

Comment:
Client secret may be cached so this search is slightly less exact - capturing both cached/non-cached log messages